package com.estudo.arquivos.poo.corepoo.exception14.runtimeexception;

public class ArithmeticException {
    public static void main(String[] args) {
        int num1 = 10;
        int num2 = 0;

        int resultado = num1 / num2;

        System.out.println(resultado);
    }

    // Arithmeticexception ocorre quando uma operação aritimetica inválida acontece
}
